evap/evaluation/tests/test_views.py

```diff
@@ -246,3 +248,51 @@ def test_notebook(self):
 
         user.refresh_from_db()
         self.assertEqual(user.notes, self.note)
+
+
+class TestResetEvaluation(WebTestStaffMode):
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.manager = make_manager()
+        cls.semester = baker.make(Semester, results_are_archived=True)
+
+    def reset_from_x_to_new(self, x, assertion):
+        evaluation = baker.make(Evaluation, state=x, course__semester=self.semester)
+
+        semester_overview_page = self.app.get(f"/staff/semester/{self.semester.pk}", user=self.manager, status=200)
+
+        form = semester_overview_page.forms["evaluation_operation_form"]
+
+        form["evaluation"] = [evaluation.pk]
+
+        confirmation_page = form.submit(name="target_state", value=str(Evaluation.State.NEW))
+        confirmation_form = confirmation_page.forms["evaluation-operation-form"]
+        confirmation_form.submit()
+
+        assertion(Evaluation.objects.get(pk=evaluation.pk))
+
+    def test_reset_to_new(self):
+        invalid_start_states = [Evaluation.State.NEW, Evaluation.State.PUBLISHED]
+
+        valid_start_states = [
+            Evaluation.State.PREPARED,
+            Evaluation.State.EDITOR_APPROVED,
+            Evaluation.State.APPROVED,
+            # TODO: allow those states:
+            # Evaluation.State.IN_EVALUATION,
+            # Evaluation.State.EVALUATED,
+            # Evaluation.State.REVIEWED
```

Comment:
CI complains about "TODO" - let's remove these for now and keep this PR unrelated to your other one :)